components/boxLink/BoxLink.js

```diff
@@ -0,0 +1,52 @@
+import { getString } from '../../lib/richText';
+import {
+  BoxLinkContainer,
+  EllipseContainer,
+  Ellipse,
+  Title,
+  Description,
+  MediumArrow,
+  LongArrow,
+  InnerContentContainer,
+} from './BoxLink.styles';
+import { icons } from '../../style/icons';
+import PageLink from '../pageLink/PageLink';
+import { breakpointsObj } from '../../lib/responsive';
+import MediaQuery from 'react-responsive';
+export default function BoxLink({ boxLinkData }) {
+  const {
+    box_link_title: boxLinkTitle,
+    box_link_description: boxLinkDescription,
+    box_link_link: boxLinkLink,
+  } = boxLinkData;
+
+  return (
+    <>
```

Comment:
don't think empty tag is not necessary here

Reply:
I think you mean don't think empty tag is necessary hehe